ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -31,82 +31,6 @@ resources:
       memory: 4000Mi
 test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
 tests:
-- as: style-checks
-  commands: .openshift-ci/dispatch.sh style-checks
-  container:
-    clone: true
-    from: root
-  secrets:
-  - mount_path: /tmp/vault/stackrox-stackrox-base
-    name: stackrox-stackrox-base
-- as: policy-checks
-  steps:
-    test:
-    - ref: stackrox-initial
-- as: mitre-bundles-checks
```

Comment:
@janisz I can't spot anything that mentions `mitre` in stackrox/stackrox `.github` directory. Was this lost or was it intentional to drop it?

Reply:
mitre is disabled so I didn't migrate it https://github.com/stackrox/stackrox/blob/fc8b4f76e2fea69d05970553ea3f1b7c097752d6/scripts/ci/jobs/mitre-bundles-checks.sh#L13-L14

Reply:
Thanks.